src/styles/global.css

```diff
@@ -1,8 +1,42 @@
+/* Import Roboto fonts from google */
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 :root {
     --MHP-green: #ccf381;
     --MHP-orange: #f2a252;
     --MHP-purple: #4831df;
```

Comment:
Not from this PR, but nitpick: According to the image I found on the drive, this should be `#4831d4`

Reply:
Haha thanks!